04a-run_ica.py

```diff
@@ -110,8 +110,17 @@ def run_ica(subject, session=None):
     msg = f'Running ICA for {kind}'
     logger.info(gen_log_message(message=msg, step=4, subject=subject,
                                 session=session))
-    ica = ICA(method='fastica', random_state=config.random_state,
-              n_components=n_components[kind])
+
+    if config.ica_algorithm == 'picard':
+        fit_params = dict(fastica_it=5)
+    elif config.ica_algorithm == 'extended_infomax':
+        fit_params = dict(extended=True)
+    elif config.ica_algorithm == 'fastica':
+        fit_params = None
```

Comment:
else error?

Reply:
No need to error, because the input values should have been sanitized at this point already:

https://github.com/mne-tools/mne-study-template/pull/106/files#diff-fdf313ed2303059c44a9ebfffbef8eebR807-R811

Reply:
Hmm link doesn't work properly… anyway, in `config.py` we do:

```python
if use_ica and ica_algorithm not in ('picard', 'fastica', 'extended_infomax'):
     msg = (f"Invalid ICA algorithm requested. Valid values for ica_algorithm "
            f"are: 'picard', 'fastica', and 'extended_infomax', but received "
            f"{ica_algorithm}.")
     raise ValueError(msg)
```